### DISCUSSION TITLE
.Net4.6  wpf cefsharp load some website is 400 error

### DISCUSSION THREAD

**Discussion by sunkejava** [cefsharp/CefSharp#4304] [General]
- **What version of the product are you using?**
    - Please only create an issue if you can reproduce the problem with version 106.0.260 or greater.
    - What version are you using? Nuget? CI Nuget? build from a branch? If so please link to the relevant commit.
	- Please include the exact version number you are using e.g. 106.0.260 (no ambiguous statements like `Latest from Nuget`)
     Nuget  every version (88.2.90.0~107.1.90) is also have the problem 
- **What architecture x86 or x64?**
    <x86/x64/arm64>
	x86 | x64
- **What version of .Net?**
    <.Net 4.x/.Net Core 3.1/.Net 5.0>
    .Net4.6.1
- **On what operating system?**
    <Win7/Win8.1/Win10/Win11>
    Win10
- **Are you using `WinForms`, `WPF` or `OffScreen`?**
    <WinForms/WPF/OffScreen>
    WPF
- **What steps will reproduce the problem?**
    Load this Web Site [https://ybj.hubei.gov.cn/hubeiHallSt/web/hallEnter/#/unitLogin](url) is Error,web control is visible 400 code
     just use cefsharp have this problem ,use cef's cefclient is run ok . the website is loaded normally.

**Comment by amaitland** [comment]
It's unlikely this is a bug. If the website loads in CEF then you need to consider the possibility that the website is deliberately blocking CefSharp. 

You can try something like the following https://github.com/cefsharp/CefSharp/discussions/4263#discussioncomment-3844085 to make it harder to detect the website is running in CefSharp.

**Reply by sunkejava** [reply]
amazing, that's successful,thank you help!
